django_extensions/management/commands/migrationstate.py

```diff
@@ -0,0 +1,171 @@
+import json
+import sys
+
+from django.apps import apps
+from django.core.management import call_command
+from django.core.management.base import \
+    BaseCommand, CommandError, CommandParser
```

Comment:
I would like to have this on a single line. Our `max-line-length` is quite high and honestly I find those `\`'s ugly :-)